src/polys/univariate.rs

```
@@ -35,6 +34,9 @@ impl<Scalar: PrimeField> UniPoly<Scalar> {
   /// Given evaluation points at consecutive integers starting from 0,
   /// this function interpolates the unique polynomial of degree `n-1`
   /// using Gaussian elimination.
+  ///
+  /// # Panics
```

Could we return an error type here?